### DISCUSSION TITLE
[Suggestions] Mage Guild screen, closed Library scrolls should not be displayed when the Library is not build

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3604] [Ideas]
In the Wizard's Castle, when the Library is not build, closed Library scrolls should not be displayed in the Mage Guild screen.

In original game:

![Castle screen mage guild level 1 H2 005](https://user-images.githubusercontent.com/43583994/96758577-3b605f00-13d7-11eb-82f7-b837ea8a9bc0.png)

In fheroes2:

![Castle screen guild level 1 fh2 005](https://user-images.githubusercontent.com/43583994/96758599-43200380-13d7-11eb-93c1-746ca1f692cd.png)

**Comment by idshibanov** [comment]
I actually think it's better to keep the additional scrolls. Similar logic to magic guild level - you see that there's something there to unlock before you build it.

**Comment by LeHerosInconnu** [comment]
Hi @idshibanov,

> I actually think it's better to keep the additional scrolls. Similar logic to magic guild level - you see that there's something there to unlock before you build it.

In the case of the Mage Guild, the building is already built, the closed scrolls are part of the same building.
For the Library, it's a different building, it has not yet been built but the closed scrolls are already displayed.
It is not logical.

Also if closed scrolls of the Library are displayed, they should be centered vertically in relation to open scrolls.

I am rather mixed, on the one hand it gives information to the player on the other hand it is not logical.
What do you think?

In fheroes2:

![Wizard library closed scroll fh2 01](https://user-images.githubusercontent.com/43583994/96867135-9d1ed880-146c-11eb-8e77-2fe427cf7207.png)

**Comment by idshibanov** [comment]
> For the Library, it's a different building, it has not yet been built but the closed scrolls are already displayed.
> It is not logical.

To me it is logical. Building Library doesn't add scrolls it **unlocks the spells**. To access this screen you would have at least a level 1 magic guild, so buying library will already unlock something. Makes perfect sense.

> Also if closed scrolls of the Library are displayed, they should be centered vertically in relation to open scrolls.

This is a different issue and I agree, should be corrected.

**Comment by LeHerosInconnu** [comment]
@idshibanov.

> > For the Library, it's a different building, it has not yet been built but the closed scrolls are already displayed.
> > It is not logical.
> 
> To me it is logical. Building Library doesn't add scrolls it **unlocks the spells**. To access this screen you would have at least a level 1 magic guild, so buying library will already unlock something. Makes perfect sense.
> 
> > Also if closed scrolls of the Library are displayed, they should be centered vertically in relation to open scrolls.
> 
> This is a different issue and I agree, should be corrected.

Okay, I have nothing against the fact that closed Library scrolls can be displayed then.

**Comment by ihhub** [comment]
Moving this issue to Discussion section.
